test/Microsoft.ApplicationInsights.AspNetCore.Tests/JavaScript/ApplicationInsightsJavaScriptTest.cs

```diff
@@ -11,19 +13,21 @@
 
     public static class ApplicationInsightsJavaScriptTest
     {
+        private static JavaScriptEncoder encoder = JavaScriptEncoder.Create(new UnicodeRange[] { UnicodeRanges.BasicLatin });
```

Comment:
That's not what this does.
It is saying anything which is not BasicLatin characters will get encoded, e.g. "<" becomes "\u003c" to prevent cross site scripting.
For the unit tests I am just setting a simple encoding.  For a live ASP.NET Core web application the framework will be automatically configured to some sensible defaults and users can always override the framework settings.